pennylane/ops/qubit.py

```diff
@@ -1662,7 +1754,7 @@ def __init__(self, *params, control_wires=None, wires=None, control_values=None,
         target_dim = 2 ** len(wires)
         if len(U) != target_dim:
             raise ValueError(f"Input unitary must be of shape {(target_dim, target_dim)}")
-
+        # Need to keep track if separate
```

Comment:
What does this mean?

Reply:
Woops, leftover comment. Originally I had to add `self._control_wires` since this op didn't keep track of its control wires. However, I just rely on `UnitaryOperation`'s adjoint instead so we don't need to keep track. I'll remove the comment.